libs/app-shell/src/lib/api/error/default-error-handler.service.ts

```diff
@@ -48,9 +48,10 @@ export class DefaultErrorHandlerService implements ErrorHandler, OnDestroy {
                 @Inject(ERROR_FORMATTER) private formatter: ErrorFormatter,
                 @Inject(ERROR_NOTIFIERS) @Optional() private notifiers: ErrorNotifier[]) {
 
-        this.topics.defineTopic({
-            prefix: 'error:', eventType: EventType.DURABLE,
-            name: TOPIC_ERROR_EVENT, shared: true
+        this.topics.set({
```

Comment:
We should leave `defineTopic`. we are not setting topic. 

The purpose of this is that you want to `create topic definition` which is later used to create subscribers / publishers. so the defineTopic is the correct one. 

If topic definition does not exists then you cant really create some random topics.